docs/source/get_started/implementers_best_practices.rst

```diff
@@ -0,0 +1,135 @@
+Securing the access to that file rests with the implementation. We recommend setting strict access rights on the file
+and ensuring it cannot be accessed by unauthorized people.
+
+
+Logs
+====
+
+While having the logging enabled is a good thing, due to the ability to audit actions and possible server failures, it
+must be taken into consideration, that logs can use a considerable amount of disk space. MOTECH allows to control
+the logging levels via property file (log4j.properties) and Admin UI. Available levels, in the decreasing order of
+importance, are: FATAL, ERROR, WARN, INFO, DEBUG and TRACE. It is also possible to set up logging ALL or disable logging
+at all, by setting OFF. Setting up logging for certain level will log all the statements of that level, plus all statements
+with the higher level. It means that setting up the logging for WARN level, will log all WARN, ERROR and FATAL statements.
+
+The loggers always have name. In MOTECH, the name of a logger corresponds to the class name it logs from, however for the logs coming
+from the external libraries, the convention for logger names can differ. If you want to set up logging for external libraries,
+please check their website or documentation for logger names. There's one logger that has got a special function and its name is **root**.
+If you have not explicitly provided different level for a logger, then this level is used. Please be cautious when changing
+the root logger level, as the amount of loggers in MOTECH and its libraries is huge. The default level is ERROR. We do
+no recommend going lower than WARN for the root logger. If you need more detailed logs, set up lower log levels for
+concrete loggers.
+
+When developing your own modules to use with MOTECH, we recommend that you remember to set up proper logging in the code.
+This helps to debug issues and find problems faster. Try to use proper logger levels when logging information.
+
+
+Data backup
+===========
+
+MOTECH does not provide any tools or support for data backup out of the box. Taking care of performing regular data backups
+lies with the implementations. Of course most of the data is kept in the database. Please remember, that MOTECH uses
+two databases. One of them is managed by the Motech Data Services module (the main persistence layer in MOTECH). Another
+database belongs to the Quartz scheduler and contains scheduler schema and scheduled jobs data. The most popular way
+to backup data are SQL dumps, that are complete snapshots of the current state of the database. Such snapshots can
+be used to restore database from scratch at any time. The creation of such backups can be easily automatized, by having
+a script that runs periodically and creates necessary snapshots.
+
+Besides databases, MOTECH stores some data in property files. The bootstrap configuration, containing database and ActiveMQ
```

Comment:
settings can be stored in the database and boostrap config can be provided as a system variable. In general, Motech should be able to operate with a volatile disk system